main/main.cpp

```diff
@@ -19,12 +19,14 @@
 #include "iassert.hpp"
 #include "lgraph.hpp"
 #include "replxx.hxx"
+
+#include "meta_api.hpp"
+#include "../eprp/eprp_var.hpp"
```

Comment:
Do not do relative path includes. eprp_var.hpp should be in the include path
#include "eprp_var.hpp"